site/content/integrate/apps/functionality/kv-store.md

```diff
@@ -4,19 +4,99 @@ description: Use the per-App key-value data store
 weight: 20
 ---
 The Apps framework provides access to a simple key-value store for App-specific data.
-Keys are simple strings and values are any type that can be marshalled to JSON.
+Keys are simple strings and values are any data that can be expressed in JSON.
 
 {{<note "Note:">}}
-It is recommended to use a [driver]({{<ref "/integrate/apps/drivers">}}) when using the key-value store.
+It is not recommended to store App configuration in the key-value store as an access token, either a user or bot token, is required to access it.
 {{</note>}}
 
-There are 3 operations that can be performed against the key-value store:
+There are three operations that can be performed against the key-value store:
 
-1. Get the value of a key (`KVGet`)
-2. Set the value of a key (`KVSet`)
-3. Delete a key (`KVDelete`)
+1. Get the value of a key
+2. Set the value of a key
+3. Delete a key
 
-For example, using the [Golang driver]({{<ref "/integrate/apps/drivers/golang">}}) to store a string value would look like the following:
+The key-value store can be accessed in two ways:
+
+- Using HTTP REST endpoints
+- Using a [driver]({{<ref "/integrate/apps/drivers">}})
```

Comment:
The driver is technically using HTTP endpoints, so this is a little confusing. It seems like we're saying it's two completely separate concepts.

```suggestion
- Using HTTP REST endpoints directly
- Using a [driver]({{<ref "/integrate/apps/drivers">}})
```

Reply:
They are to an implementor. One's easy, the other's low-level.
@asatkinson Please confirm if we should have a driver concept at all given the REST endpoints we've been documenting.

Reply:
I agree with these points!